### ISSUE TITLE
Build fails due to missing README

### ISSUE THREAD

**Issue by mightyiam** [slacka/WinUSB#86]
```
cp: cannot stat 'README': No such file or directory                                 
dh_installdocs: cp --reflink=auto -a README debian/winusb/usr/share/doc/winusb returned exit code 1
debian/rules:74: recipe for target 'binary-arch' failed                                                                                                                            
make: *** [binary-arch] Error 2                                                                                                                                                    
dpkg-buildpackage: error: fakeroot debian/rules binary gave error exit status 2
```
Here is the full [output.txt](https://github.com/slacka/WinUSB/files/739849/output.txt)

**Comment by Lin-Buo-Ren** [comment]
Thanks for the report, this is an regression caused by commit 143dc, it has been fixed since commit b846c90b.

**Comment by mightyiam** [comment]
Cool. Thank you, @Lin-Buo-Ren.
